pulp_container/constants.py

```diff
@@ -29,6 +30,7 @@
             MEDIA_TYPE.MANIFEST_LIST,
             MEDIA_TYPE.INDEX_OCI,
             MEDIA_TYPE.MANIFEST_OCI,
+            MEDIA_TYPE.OCI_EMPTY_JSON,
```

Comment:
this does not go into the accept headers. only manifests mediatype types go. this is  blob instead.